### PR TITLE
Improving the Link-State 2015 Twitter Card

### PR DESCRIPTION
**Pull request by thomasrussellmurphy**
Having the date in the description and trying to get the image to work using a full link in the meta tag (still with programmatic retrieval of the path).

### CODE REVIEW

File: app/views/acm/conference/2015.haml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #{meta_tag "summary", :name => "twitter:card"}
 #{meta_tag "@cwruacm", :name => "twitter:site"}
 #{meta_tag "EECS @ CWRU -- Link-State 2015 Conference", :name => "twitter:title"}
-#{meta_tag "The fourth annual student-run tech conference at Case Western Reserve University", :name => "twitter:description"}
-#{meta_tag "#{image_path 'LinkState175px.png'}", :name => "twitter:image"}
+#{meta_tag "The fourth annual student-run tech conference at Case Western Reserve University taking place all day on Saturday October 3rd, 2015.", :name => "twitter:description"}
+#{meta_tag "http://acm.case.edu" + "#{image_path 'LinkState175px.png'}", :name => "twitter:image"}
```
Comment (ajm188) [inline app/views/acm/conference/2015.haml]: I think you can do `image_url` then you don't have to hard code the host here.

**Comment by thomasrussellmurphy** [comment]
`image_url` is not an available Padrino function.

**Comment by ajm188** [comment]
Dang. :heart: rails.

:+1: then.

**Comment by thomasrussellmurphy** [comment]
Deployed. New image link indeed works, if it's a bit cropped by Twitter. . .

**Pull request merged**